src/Snowflaqe.fsproj

```diff
@@ -7,13 +7,10 @@
     <PackAsTool>true</PackAsTool>
     <IsPackable>true</IsPackable>
     <RollForward>Major</RollForward>
-    <TargetFramework>net5.0</TargetFramework>
-    <Version>1.24.0</Version>
-    <PackageReleaseNotes>
-      Fix copyLocalLockFileAssemblies option.
-      Update Fable.Remoting.Json and SimpleJson to latest.
-      Lower required FSharp.Core version to v4.7.2
-    </PackageReleaseNotes>
+    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
Using `<TargetFramework>net5.0</TargetFramework>` should suffice, please revert. Unless there is some issue with `net5.0`?

Reply:
We need `netstandard2.0` for MSBuild running on .NET Framework (used in Visual Studio)
And `netcoreapp3.1` is used for MSBuild Core

Reply:
Alright, I see. That's fine then